rm-datasource/src/main/java/io/seata/rm/datasource/sql/druid/BaseMySQLRecognizer.java

```diff
@@ -52,7 +53,8 @@ public boolean visit(SQLVariantRefExpr x) {
                         oneParamValues.stream().forEach(t -> paramAppenderList.add(new ArrayList<>()));
                     }
                     for (int i = 0; i < oneParamValues.size(); i++) {
-                        paramAppenderList.get(i).add(oneParamValues.get(i));
+                        Object o = oneParamValues.get(i);
+                        paramAppenderList.get(i).add(o instanceof Null ? null : o);
```

Comment:
1.AbstractPreparedStatementProxy.setNull-->2.BaseXXXRecognizer-->3.XXXExecutor
What is the meaning of Null?
If you modify it in the second step, does Null lose its meaning?

Reply:
my mistake